lib/package/Bundle.js

```diff
@@ -218,9 +237,16 @@ function Bundle(config, cb) {
   //rewrite the source to be filesystem
   var bundle = this;
 
-  bundle.excluded = config.excluded;
-
+  this.xPathName = bundleType.getXPathName(config.source.bundleType);
+  this.bundleTypeName = config.source.bundleType;
+  this.excluded = config.excluded;
```

Comment:
there was a bug here on master branch. Excluded options were not being copied to the bundle which was causing the linter and test suite to fail.